python_modules/dagster/dagster_tests/daemon_tests/test_backfill.py

```diff
@@ -701,25 +743,29 @@ def test_pure_asset_backfill(instance, workspace_context, external_repo):
     wait_for_all_runs_to_finish(instance, timeout=30)
 
     assert instance.get_runs_count() == 3
-    runs = reversed(instance.get_runs())
+    runs = reversed(list(instance.get_runs()))
     for run in runs:
         assert run.tags[BACKFILL_ID_TAG] == "backfill_with_asset_selection"
         assert step_succeeded(instance, run, "foo")
         assert step_succeeded(instance, run, "reusable")
         assert step_succeeded(instance, run, "bar")
     # selected
     for asset_key in asset_selection:
-        assert len(instance.run_ids_for_asset_key(asset_key)) == 3
+        assert len(list(instance.run_ids_for_asset_key(asset_key))) == 3
```

Comment:
Prob negligible performance change but there is `len_iter` for these cases:

https://github.com/dagster-io/dagster/blob/master/python_modules/dagster/dagster/_utils/__init__.py#L731-L734

Reply:
oo nice